src/main/java/com/yahoo/bullet/parsing/Query.java

```diff
@@ -39,12 +41,16 @@
     @Expose
     private List<PostAggregation> postAggregations;
 
+    public static final BulletError NO_DELIMITERS = makeError("Projection fields cannot contain delimiters.",
```

Comment:
Is this for BQL? I guess it can break aggregations etc too to have a field with a '.'